package.json

```diff
@@ -41,6 +41,7 @@
     "eslint-config-next": "11.1.2",
     "jest": "^27.4.4",
     "react-test-renderer": "^17.0.2",
-    "typescript": "4.4.4"
+    "typescript": "4.4.4",
+    "webpack": "^5.67.0"
```

Comment:
Why was this added btw?

Reply:
I was unsuccessful running the VUI with SpecialText inside react-ui's `main` branch. I was getting the `fetchcompilewasmtemplateplugin is not a constructor` error which I believe is related to webpack. To resolve this issue, I packaged SpecialText with react-ui's `vui-export`. I believe Jianjia (the creator of the branch) installed some plugins which resolved this webpack issue, hence why we add webpack here.

Reply:
Ah that error is related to workers.ts no worries